test/skimNtuple_HHbtag.cpp

```diff
@@ -5214,21 +5197,34 @@ int main (int argc, char** argv)
 		  theSmallTree.m_METy_taudown_DM10 = vMET_shift_tes.second.at(2).Y();
 		  theSmallTree.m_METy_taudown_DM11 = vMET_shift_tes.second.at(3).Y();
 
-		  // Shifted MET for EES
-		  theSmallTree.m_METx_eleup_DM0 = vMET_shift_ees.first.at(0).X();
-		  theSmallTree.m_METx_eleup_DM1 = vMET_shift_ees.first.at(1).X();
+		  // Shifted MET for EES (electrons reconstructed as fake taus)
+		  theSmallTree.m_METx_eleup_DM0 = vMET_shift_ees_fakes.first.at(0).X();
+		  theSmallTree.m_METx_eleup_DM1 = vMET_shift_ees_fakes.first.at(1).X();
 
-		  theSmallTree.m_METy_eleup_DM0 = vMET_shift_ees.first.at(0).Y();
-		  theSmallTree.m_METy_eleup_DM1 = vMET_shift_ees.first.at(1).Y();
+		  theSmallTree.m_METy_eleup_DM0 = vMET_shift_ees_fakes.first.at(0).Y();
+		  theSmallTree.m_METy_eleup_DM1 = vMET_shift_ees_fakes.first.at(1).Y();
 
-		  theSmallTree.m_METx_eledown_DM0 = vMET_shift_ees.second.at(0).X();
-		  theSmallTree.m_METx_eledown_DM1 = vMET_shift_ees.second.at(1).X();
+		  theSmallTree.m_METx_eledown_DM0 = vMET_shift_ees_fakes.second.at(0).X();
+		  theSmallTree.m_METx_eledown_DM1 = vMET_shift_ees_fakes.second.at(1).X();
+
+		  theSmallTree.m_METy_eledown_DM0 = vMET_shift_ees_fakes.second.at(0).Y();
+		  theSmallTree.m_METy_eledown_DM1 = vMET_shift_ees_fakes.second.at(1).Y();
+
+		  // Shifted MET for EES (real electrons not in tautau)
+		  auto vMET_shift_electrons_scale = getShiftedMET_electrons(vMET, theBigTree, pairType, true);
```

Comment:
(open question) do we really want to decorrelate the EES MET shift from electrons from the one from e faking taus?

Reply:
Thanks for raising this point.
The two sets of corrections come from the recommendation of different POGs (Tau vs E/Gamma).
Given that genuine and electrons and electron faking taus are different objects, it feels reasonable to decorrelate their uncertainties.
In a similar but more extreme way, we could also wonder whether we should correlate MET shifts for electrons and muons, for example, and intuitively my answer would be no.

Reply:
correlating electrons and muons would be a different issue; here we're talking about a *single correction* (EES shifts) applied to different objects

Reply:
I agree with Bruno, the way I understand this those are two different corrections (+ uncertainties)... it's a bit unfortunately that we call them both EES, I guess the e faking a tau one should actually be called [FES](https://github.com/cms-tau-pog/TauIDSFs?tab=readme-ov-file#eta---dm-dependent-e---tau-fake-energy-scale)?